test/common/tests.go

```diff
@@ -7,7 +7,15 @@ var (
 		{"Verify CRD Exists", TestIntegreatlyCRDExists},
 	}
 
+	AFTER_INSTALL_PREREQUISITES = []TestCase{
+		// Prerequisite checks, to be executed before tests
+		{"Setup Testing IDP", SetupTestingIDP},
```

Comment:
Not sure about this approach, doing so if the setup of the testing idp fails, then none of the other tests will be executed even if not all tests depend on the testing idp, this is breaking the principle of having [Independent](https://github.com/integr8ly/integreatly-operator/blob/master/test/BEST_PRACTICES.md#independent) tests

Reply:
makes sense, good catch :+1: